devicedb/deb/debian/devicedb.service

```diff
@@ -11,4 +15,4 @@ ExecStart=/usr/bin/devicedb start -conf /etc/pelion/devicedb.yaml
 Type=simple
 
 [Install]
-WantedBy=multi-user.target
+WantedBy=pelion.service
```

Comment:
I'm not sure you want to change `WantedBy`  The `Install` section is [roughly equivalent to the old sysv init run levels](https://unix.stackexchange.com/a/506374/333250) and `multi-user.target` represents the old runlevel 5.  I think we still want that.